src/spikeinterface/core/tests/test_whitematterrecordingextractor.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+import numpy as np
+
+from spikeinterface.extractors import WhiteMatterRecordingExtractor
+from spikeinterface.core.numpyextractors import NumpyRecording
+
+
+def test_WhiteMatterRecordingExtractor(create_cache_folder):
+    cache_folder = create_cache_folder
+    num_seg = 2
+    num_channels = 3
+    num_samples = 30
+    sampling_frequency = 10000
+    dtype = "int16"
+
+    file_paths = [cache_folder / f"test_WhiteMatterRecordingExtractor_{i}.raw" for i in range(num_seg)]
+    for i in range(num_seg):
+        np.memmap(file_paths[i], dtype=dtype, mode="w+", shape=(num_samples, num_channels))
+
+    rec = WhiteMatterRecordingExtractor(
+        file_paths=file_paths,
+        sampling_frequency=sampling_frequency,
+        num_channels=num_channels,
+    )
```

Comment:
How is this working? You are writing the file without an offset, and the `WhiteMatterRecording` excpects an 8-bit offset..

I would remove this test and just keep the other two

Reply:
Done!